cmd/admission-webhook/main.go

```diff
@@ -16,28 +16,34 @@ package main
 import (
 	"flag"
 	"os"
+	"runtime"
 	"time"
 
-	"github.com/openshift/generic-admission-server/pkg/cmd"
+	"github.com/openshift/generic-admission-server/pkg/apiserver"
+	"github.com/openshift/generic-admission-server/pkg/cmd/server"
+	"k8s.io/klog"
+
 	"github.com/pingcap/tidb-operator/pkg/features"
 	"github.com/pingcap/tidb-operator/pkg/version"
 	"github.com/pingcap/tidb-operator/pkg/webhook"
+	genericapiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/component-base/logs"
 )
 
 var (
+	flagset                  *flag.FlagSet
 	printVersion             bool
 	extraServiceAccounts     string
 	evictRegionLeaderTimeout time.Duration
 )
 
 func init() {
+	flagset = flag.NewFlagSet("tidb-admission-webhook", flag.ExitOnError)
```

Comment:
Do we need to introduce the flagset? If so, I think we should use only one of flagset or the standard library flag.

Reply:
updated, this `flagset` is not necessary and removed now.